src/components/Tooltip/TooltipContent.tsx

```diff
@@ -0,0 +1,42 @@
+/* tslint:disable:no-unused-variable */
+import * as React from 'react';
+/* tslint:enable:no-unused-variable */
+import { BaseComponent } from '../../common/BaseComponent';
+import { ITooltipProps } from './Tooltip.Props';
+import { ITooltipState } from './Tooltip';
+import { getId } from '../../Utilities';
+import './Tooltip.scss';
+
+export class TooltipContent extends BaseComponent<ITooltipProps, ITooltipState> {
```

Comment:
I can consolidate. Wasn't sure what best practice was. Had combined content in the teaching bubble and someone broke it out into a separate content partial.